### PR TITLE
update: adds testing for CELL, INFO, N, SHEET and SHEETS

### LINKED ISSUE #515: Please test and create docs for N, SHEETS, INFO and CELL
Note that:

* INFO is not implemented at all. You could add Rust tests, cheking argument number and that it returns N/IMPL!.
You could also think about what we should implement
* SHEETS is only implemented when it has no arguments
* CELL has a bunch of things that we do not impement. I will implement some of them like COLOR later on, but for now we are compatible with Excl for the Web. you could add the tests manually. (in this case it return #VALUE!)

Good luck!

### PR DESCRIPTION
**Pull request by elsaminsut**
This PR adds testing for the CELL, INFO, N, SHEET and SHEETS functions. These include both xlsx tests with edge cases and unit tests to check the number of arguments taken by the function. It also includes an update to the main Information documentation page and their individual pages to show them as now available.

This PR closes #515

(See failing edge cases in #533)

### CODE REVIEW

File: base/src/test/test_cell_info_n_sheets [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    assert_eq!(model._get_text("A3"), *"#N/IMPL!");
+    assert_eq!(model._get_text("A4"), *"#ERROR!");
+
+    assert_eq!(model._get_text("A5"), *"1");
+    assert_eq!(model._get_text("A6"), *"#ERROR!");
+    assert_eq!(model._get_text("A7"), *"#ERROR!");
+
+    assert_eq!(model._get_text("A8"), *"1");
+    assert_eq!(model._get_text("A9"), *"#N/IMPL!");
+}
```
Comment (nhatcher) [inline base/src/test/test_cell_info_n_sheets]: Missing new line

Comment (elsaminsut) [reply base/src/test/test_cell_info_n_sheets]: fixed!

**Comment by nhatcher** [comment]
> You can see the failing tests split per function in https://github.com/ironcalc/IronCalc/issues/533

Thanks. In this PR can you split the test case in different files also?
Ideally we would have 1 file per functions.

**Comment by elsaminsut** [comment]
> > You can see the failing tests split per function in #533
> 
> Thanks. In this PR can you split the test case in different files also? Ideally we would have 1 file per functions.

Fixed!

**Review by nhatcher — APPROVED** [review]
